pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -23,17 +23,29 @@ let
       fi
       mkdir -p "$out/bin"
 
-      for path in ${stdenv.lib.concatStringsSep " " paths}; do
-        if [ -d "$path/bin" ]; then
-          cd "$path/bin"
-          for prg in *; do
-            if [ -f "$prg" ]; then
-              rm -f "$out/bin/$prg"
-              makeWrapper "$path/bin/$prg" "$out/bin/$prg" --set PYTHONHOME "$out"
-            fi
+      # Generate new wrapper scripts for the all the executables found in our
```

Comment:
It is necessary --- before, this code would try to wrap anything at top level in `$path/bin`, including non-executable files (in this case, `.pyc` files) and directories.